services/asset-manager/Makefile

```diff
@@ -1,6 +1,7 @@
-asset-manager: $(GOVUK_ROOT_DIR)/$@
+asset-manager: $(GOVUK_ROOT_DIR)/asset-manager
 	$(GOVUK_DOCKER) compose build $@-lite
-	$(GOVUK_DOCKER) compose run $@-lite rbenv install -s && gem install --conservative bundler
+	$(GOVUK_DOCKER) compose run $@-lite rbenv install -s
+	$(GOVUK_DOCKER) compose run $@-lite gem install --conservative bundler -v "~>1.0"
```

Comment:
as per the civil service code, I'm not sure we're allowed to show preference for a conservative bundler.

Reply:
How about `--liberal bundler`?

Reply:
😑